### PR TITLE
Amb 420 update helloworld spec text

### PR DESCRIPTION
**Pull request by SachinPath**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [ ] Dev
* [ ] Test
* [x] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: specification/hello-world.yaml [hunk 1]
```diff
@@ -1,62 +1,94 @@
...
+
+    Our open-access APIs are generally RESTful APIs. For more details on how to access them, see our [open-access REST API tutorial](https://digital.nhs.uk/developer/guides-and-documentation/tutorials/open-access-rest-api-tutorial).
 
-    ## Environments and Testing
-    This API is for learning purposes and is only available in our sandbox environment. The base URL for all endpoints is:
-    - `https://sandbox.api.service.nhs.uk/hello/`
+    ### Application-restricted authentication 
+    Use this authentication method to access the application endpoint. This access mode is [application-restricted](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis), meaning we authenticate the calling application but not the end user.
+    You can use this access mode as follows:
+     - unattended (end user not present), for example as part of a back end process to check NHS numbers for data flowing from one system to another
```
Comment (artronics) [inline specification/hello-world.yaml:46]: @SachinPath Hello World doesn't have `unattended` authentication. You can remove this line.

File: specification/hello-world.yaml [hunk 2]
```diff
@@ -1,62 +1,94 @@
...
+     - attended (end user present) - in which case, you must ensure the end user is authenticated and suitably authorised locally by the calling application
+    To use this access mode, use the following security pattern:
+     - [Application-restricted RESTful API - signed JWT authentication](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation/application-restricted-restful-apis-signed-jwt-authentication)
```
Comment (artronics) [inline specification/hello-world.yaml:49]: AFAIK "signed JWT authentication" is for unattended access. For HelloWorld "Application Restriced" endpoint we only need an apikey to authenticate.

File: specification/hello-world.yaml (hunk 1 above)
Comment (SachinPath) [reply specification/hello-world.yaml]: Removed unattended authentication.

File: specification/hello-world.yaml (hunk 2 above)
Comment (SachinPath) [reply specification/hello-world.yaml]: Updated the link to apikey authentication
https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation/application-restricted-restful-apis-api-key-authentication

**Review by artronics — DISMISSED** [review]

**Review by mick-sch — APPROVED** [review]
All OK.

**Review by mick-sch — APPROVED** [review]
Looks good.

**Pull request merged**